Kudu.Core.Test/Infrastructure/EnvironmentFacts.cs

```diff
@@ -125,7 +125,8 @@ public void SSHKeyPathCreatesDirectoryIfItDoesNotExist()
             string sshKeyPath = null,
             string scriptPath = null,
             string nodeModulesPath = null,
-            string dataPath = null)
+            string dataPath = null,
+            string binPath = null)
```

Comment:
v3 client loads provider using MEF, we need to tell MEF where to load those provider (dll), so binPath is required now because of MEF.

for more details, please look at [SiteExtensionManager constructor](https://github.com/projectkudu/kudu/pull/1450/files#diff-afacb573b03286d1f84a90cda5b57ddaR111)